### PR TITLE
*: bump to network-runner@v1.2.2

### PR DESCRIPTION
**Pull request by hexfusion**

### CODE REVIEW

File: rpcpb/rpc.proto [hunk 1]
```diff
@@ -1,14 +1,17 @@
-// Copied from https://github.com/ava-labs/avalanche-network-runner/blob/v1.1.0/rpcpb/rpc.proto
-// without gRPC gateway dependency
-
 syntax = "proto3";
 
-option go_package = "github.com/ava-labs/avalanche-network-runner-sdk;rpcpb";
+option go_package = "github.com/ava-labs/avalanche-network-runner;rpcpb";
```
Comment (gyuho) [inline rpcpb/rpc.proto]: Can we keep `/avalanche-network-runner-sdk`?

File: rpcpb/rpc.proto [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 
 package rpcpb;
 
+import "google/api/annotations.proto";
+
 service PingService {
   rpc Ping(PingRequest) returns (PingResponse) {
+    option (google.api.http) = {
```
Comment (gyuho) [inline rpcpb/rpc.proto]: Do we need this?

File: rpcpb/rpc.proto (hunk 1 above)
Comment (hexfusion) [reply rpcpb/rpc.proto]: yeah just saw that

File: rpcpb/rpc.proto [hunk 3]
```diff
@@ -1,14 +1,17 @@
-// Copied from https://github.com/ava-labs/avalanche-network-runner/blob/v1.1.0/rpcpb/rpc.proto
-// without gRPC gateway dependency
```
Comment (gyuho) [inline rpcpb/rpc.proto:2]: Can we keep this but update the rpc.proto link to the latest

**Review by gyuho — APPROVED** [review]

**Pull request merged**